lib/proto/proto_conn.c

```diff
@@ -191,23 +191,24 @@ proto_conn_drop(void * conn_cookie, int reason)
 }
 
 /**
- * proto_conn_create(s, sas, decr, nopfs, requirepfs, nokeepalive, K, timeo,
- *     callback_dead, cookie):
+ * proto_conn_create(s, sas, sa_b, decr, nopfs, requirepfs, nokeepalive, K,
+ *     timeo, callback_dead, cookie):
  * Create a connection with one end at ${s} and the other end connecting to
- * the target addresses ${sas}.  If ${decr} is 0, encrypt the outgoing data;
- * if ${decr} is nonzero, decrypt the incoming data.  If ${nopfs} is non-zero,
- * don't use perfect forward secrecy.  If ${requirepfs} is non-zero, drop
- * the connection if the other end tries to disable perfect forward secrecy.
- * Enable transport layer keep-alives (if applicable) on both sockets if and
- * only if ${nokeepalive} is zero.  Drop the connection if the handshake or
+ * the target addresses ${sas}.  Optionally bind to address ${sa_b}.  If
+ * ${decr} is 0, encrypt the outgoing data;  * if ${decr} is nonzero, decrypt
```

Comment:
Error in vim's reflow: it kept the initial " * " in the rewrapped text.  (That happens to me often.)

Reply:
Fixed.